corehq/apps/app_manager/suite_xml/post_process/remote_requests.py

```diff
@@ -180,6 +180,11 @@ def build_title(self):
             text=Text(locale_id=id_strings.case_search_title_translation(self.module))
         )
 
+    def build_description(self):
+        return Display(
```

Comment:
It's late for this feedback, but it also would be fine to only add a description node if there is a description, which should save you from updating all the text XML files.

Reply:
That makes sense, probably cleaner to not include it where it would be empty. And not a ton of work to go back and change. Does something like this as a sub for [line 205](https://github.com/dimagi/commcare-hq/blob/eb1efaddfd7a763768e7b0400527437f589602ff/corehq/apps/app_manager/suite_xml/post_process/remote_requests.py#L205) seem to make sense for it?
```python
description=self.build_description() if self.module.search_config.description != {} else None,
```